client/app/scripts/superdesk-vocabularies/views/vocabulary-config.html

```diff
@@ -0,0 +1,15 @@
+<div sd-shadow>
+  <ul class="pills-list">
+
+    <li ng-repeat="vocabulary in vocabularies._items">
```

Comment:
Can the list of vocabularies change? If yes, it is recommended to add `track by` here
(and if not, `vocabularies._items` should probably be one-time bound)